Refresh.GameServer/Services/DiscordStaffService.cs

```diff
@@ -0,0 +1,75 @@
+using System.Diagnostics;
+using Bunkum.Core.Services;
+using Discord;
+using Discord.Webhook;
+using NotEnoughLogs;
+using Refresh.GameServer.Configuration;
+using Refresh.GameServer.Types.UserData;
+using GameAsset = Refresh.GameServer.Types.Assets.GameAsset;
+
+namespace Refresh.GameServer.Services;
+
+public class DiscordStaffService : EndpointService
+{
+    private readonly DiscordWebhookClient? _client;
+    private readonly IntegrationConfig _config;
+    
+    private readonly string _externalUrl;
+
+    private const string NameSuffix = " (Staff)";
+    
+    private const string DefaultResultsDescription = "These are the results of the AI's best guess at deciphering the contents of the image. " +
+                                                     "Take them with a grain of salt as the AI isn't perfect.";
+
+    internal DiscordStaffService(Logger logger, GameServerConfig gameConfig, IntegrationConfig config) : base(logger)
+    {
+        this._config = config;
+        this._externalUrl = gameConfig.WebExternalUrl;
+        
+        if(config.DiscordStaffWebhookEnabled)
+            this._client = new DiscordWebhookClient(config.DiscordStaffWebhookUrl);
+    }
+    
+    private string GetAssetUrl(string hash)
+    {
+        return $"{this._externalUrl}/api/v3/assets/{hash}/image";
+    }
+    
+    private string GetAssetInfoUrl(string hash)
+    {
+        return $"{this._externalUrl}/api/v3/assets/{hash}/image";
+    }
```

Comment:
Did you mean this? The function is labelled as `AssetInfoUrl`.
```suggestion
    private string GetAssetInfoUrl(string hash)
    {
        return $"{this._externalUrl}/api/v3/assets/{hash}";
    }
```